.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+image: gitpod/workspace-python-3.8
+
+tasks:
+  - init: pip install -r requirements.txt
+    command: python main.py
```

Comment:
There is no point running the script at the start:
a) It is not a server
b) It shows an error, because the boilerplate has not defined the function

```suggestion
```

Reply:
Very true. I'll go through and remove these now.